buildscripts/provisioner-localpv/provisioner-localpv.Dockerfile

```diff
@@ -54,6 +54,14 @@ LABEL org.label-schema.build-date=$DBUILD_DATE
 LABEL org.label-schema.vcs-url=$DBUILD_REPO_URL
 LABEL org.label-schema.url=$DBUILD_SITE_URL
 
+RUN apk add --no-cache \
+    iproute2 \
+    bash \
+    curl \
+    net-tools \
+    procps \
+    ca-certificates
+
```

Comment:
@kmova, @prateekpandey14  2 packages `libc6-compat` and `mii-tools` has been removed which was not needed.

Reply:
@akhilerm can you please test the multi arch image locally once, bit concern about `libc6-compat`  that removed from the dockerfile, basically `libc6-compat` is compatibility libraries for glibc, so its good to test once

Reply:
Tested in a cluster, didn't face any issues with liveness probe or any other part.